projects/aca-playwright-shared/src/page-objects/pages/favorites.page.ts

```diff
@@ -44,8 +44,15 @@ export class FavoritesPage extends BasePage {
   public sidenav = new SidenavComponent(this.page);
   public pagination = new PaginationComponent(this.page);
   public breadcrumb = new Breadcrumb(this.page);
+  public infoDrawer = new AdfInfoDrawerComponent(this.page);
 
   async waitForPageLoad() {
     await this.page.waitForURL(`**/${FavoritesPage.pageUrl}`);
   }
+
+  async addCommentToNode(commentText: any) {
```

Comment:
Also I think this method should be a part of info drawer page so that it can be reusable in other pages, I think you already duplicated it in another page

Reply:
True, this method is now a part of infoDrawer page. Deleted it from favorites and recent-files. WIll be fixed in the next push